tutorials/time-freq/ssvep.py

```diff
@@ -0,0 +1,359 @@
+"""
+==================================
+Basic analysis of an SSVEP dataset
+==================================
+
+Example script to compute frequency spectrum and extract snr of a target frequency
+
+We use a simple example dataset with frequency tagged visual stimulation (a.k.a.
+steady state visually evoked potentials - SSVEP):
+
+N=2 participants observed checkerboards patterns inverting with a constant frequency
+of either 12Hz of 15Hz. 10 trials of 30s length each. 32ch wet EEG was recorded.
+
+Data format: BrainVision .eeg/.vhdr/.vmrk files organized according to BIDS standard.
+
+Data can be downloaded at https://osf.io/7ne6y/
+"""  # noqa: E501
+# Authors: Dominik Welke <dominik.welke@ae.mpg.de>
+#          Evgenii Kalenkovich <e.kalenkovich@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+import mne
+import numpy as np
+from mne_bids import make_bids_basename, read_raw_bids
+from scipy.stats import ttest_rel, ttest_ind
+
+###############################################################################
+# Set parameters
+# Load eeg trigger structure / event_id
```

Comment:
could just become 
`# load data`

Reply:
also, linebreaks dont seem to work in the built tutorial. does it maybe need a special character like `\n`?

Reply:
> could just become
> `# load data`

I'll collapse the two sections then.

Reply:
> also, linebreaks dont seem to work in the built tutorial. does it maybe need a special character like `\n`?

You are correct - I missed that they got concatenated. The way the paragraphs are delineated is with an empty commented string, like this:

```python
# Paragraph
#
# Another
# paragraph
```